src/events.js

```diff
@@ -11,55 +11,38 @@ const log = debug('watsonwork-weather-events');
 // event, the original annotated message and the user who sent it
 const callback = (evt, appId, info, annotation, token, cb) => {
 
-  // Retrieve the annotated message
+    // Retrieve the annotated message
   messages.message(evt.messageId, token(), (err, message) => {
-    if(err)
+    if (err)
       return;
 
-    // Ignore messages from the app itself
-    if(message.createdBy.id === appId)
+        // Ignore messages from the app itself
+    if (message.createdBy.id === appId)
       return;
 
-    // Return the extracted info, annotation, annotated message
-    // and the user who sent it
+        // Return the extracted info, annotation, annotated message
```

Comment:
Looks like another unnecessary formatting change (more of these in this pull request)